Dockerfile

```diff
@@ -1,31 +1,21 @@
-FROM ruby:latest
+FROM registry.redhat.io/ubi8/ruby-25
+
+ARG OTW_WEB_PORT=7143
```

Comment:
OTS_WEB_PORT instead of OTW_ ?

Reply:
Also, you parametrize the APP port in the Dockerfile but keep it hardcoded in the entrypoint.